Cargo.toml

```
@@ -16,10 +16,13 @@ name = "mask"
 path = "src/lib.rs"
 
 [dependencies]
-clap = "2.33.0"                                                         # https://github.com/clap-rs/clap
 colored = "1.8.0"                                                       # https://github.com/mackwic/colored
 pulldown-cmark = { version = "0.5", default-features = false }          # https://github.com/raphlinus/pulldown-cmark
 
+[dependencies.clap]																# https://github.com/clap-rs/clap
+version = "2.33.0"
+features = ["wrap_help", "color"]
```

Can we remove "color" here? It's a [default feature](https://github.com/clap-rs/clap#features-enabled-by-default) so we don't need to specify it :)

```suggestion
features = ["wrap_help"]
```

---

done 702afad :yum: